examples/lighting-app/efr32/src/Rpc.cpp

```diff
@@ -70,11 +72,8 @@ int Init()
     pw_sys_io_Init();
 
     // Start App task.
-    if (xTaskCreate(RunRpcService, "RPC_TASK", RPC_TASK_STACK_SIZE / sizeof(StackType_t), nullptr, RPC_TASK_PRIORITY,
-                    &sRpcTaskHandle) == pdPASS)
-    {
-        err = CHIP_NO_ERROR;
-    }
+    sRpcTaskHandle = xTaskCreateStatic(RunRpcService, "RPC_TASK", RPC_TASK_STACK_SIZE / sizeof(StackType_t), nullptr,
+                                       RPC_TASK_PRIORITY, sRpcTaskStack, &sRpcTaskBuffer);
```

Comment:
For future reference, `ArrayLength(sRpcTaskStack)` would have been a good idea here.

Reply:
Added this cleanup to the other RPC PR: https://github.com/project-chip/connectedhomeip/pull/5866/commits/c752eefb16138e124b362dcae69ebdd5df87b2b2